Code/Tools/AssetProcessor/native/utilities/GUIApplicationManager.cpp

```diff
@@ -73,6 +74,14 @@ namespace
     
 }
 
+ErrorCollector::~ErrorCollector()
+{
+    MessageWindow messageWindow(m_parent);
+    messageWindow.SetHeaderText("The following errors occurred during startup:");
+    messageWindow.SetMessageText(m_errorMessages);
+    messageWindow.SetTitleText("Startup Errors");
+    messageWindow.exec();
```

Comment:
will this show up even if there were no errors?

Reply:
Good catch, fixed.